src/bsk_rl/envs/general_satellite_tasking/utils/logging_config.py

```diff
@@ -0,0 +1,21 @@
+import logging
+
+sim_format = logging.Formatter(
```

Comment:
Do you think adding a color scheme would be interesting to make it easier to identify when it is INFO, DEBUG, or WARNING? Something like blue, brown, and yellow would help, especially when debugging when we have a lot of info (if it is not too much work now).

Reply:
Also, is it worth adding the option to save the log in a file? I can see some cases where it might be useful and easier to use the file than the terminal.

Reply:
Lol I was already working on color scheme when you said this, this is what I just pushed. Not shown in the pic are colors for warnings (yellow) and errors (red)
<img width="1264" alt="Screenshot 2023-12-23 at 8 46 18 AM" src="https://github.com/AVSLab/bsk_rl/assets/1545655/53e98db9-55bc-4a3b-a071-15e198901ed5">

Reply:
Was considering adding to-file logging, should be pretty easy I think.

Reply:
Excellent! The colors look nice!